spec/system/task_spec.rb

```diff
@@ -1,24 +1,24 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+  let(:project) { create(:project) }
+  let(:task) { create(:task, project_id: project.id) }
```

Comment:
TaskのFactoryに`association`を定義すると、project_idの指定が不要となるので確認してみてください。
```suggestion
  let(:task) { create(:task) }
```

Reply:
e2c7f575cd078b811eb964b7f6e88c0537cce129 で修正しました。